test/hotspot/jtreg/runtime/appcds/cacheObject/ArchivedModuleWithCustomImageTest.java

```diff
@@ -45,6 +42,7 @@
 import jdk.test.lib.compiler.CompilerUtils;
 import jdk.test.lib.process.OutputAnalyzer;
 import jdk.test.lib.process.ProcessTools;
+import jdk.test.lib.helpers.ClassFileInstaller;
```

Comment:
Includes the changes from [JDK-8263412](https://bugs.openjdk.org/browse/JDK-8263412) ( commit [e834f99](https://github.com/openjdk/jdk/commit/e834f99d1dee8070e626fd1a4657e0522a414ae7) ) on this file